mne/source_estimate.py

```diff
@@ -3231,9 +3237,18 @@ def stc_near_sensors(evoked, trans, subject, distance=0.01, mode='sum',
     _validate_type(evoked, Evoked, 'evoked')
     _validate_type(mode, str, 'mode')
     _check_option('mode', mode, ('sum', 'single', 'nearest'))
-    evoked = evoked.copy().pick_types(meg=False, ecog=True)
-    pos = np.array([ch['loc'][:3] for ch in evoked.info['chs']])
+
+    # create a copy of Evoked using either ecog or seeg
+    if src is None:
+        evoked = evoked.copy().pick_types(meg=False, ecog=True)
+    else:
+        evoked = evoked.copy().pick_types(meg=False, seeg=True)
+    # get channel positions and coord_frame transformation
+    # XXX: how to allow MNI?
+    pos = evoked._get_channel_positions()
```

Comment:
The channel positions should always be in the head coordinate frame. Then we use `trans` to get them to the MRI coordinate frame.

People with coordinates in MNI coords should just use `fsaverage` as their subject since fsaverage is in MNI space (i.e., MNI==fsaverage's MRI coordinate frame)